src/github.com/stellar/federation/request_handler.go

```diff
@@ -29,31 +28,30 @@ func (rh *RequestHandler) Main(w http.ResponseWriter, r *http.Request) {
 }
 
 func (rh *RequestHandler) MakeDatabaseRequest(query string, w http.ResponseWriter, r *http.Request) {
-	record := Record{}
+	record := FedRecord{}
 
-	q := r.URL.Query().Get("q")
+	stellarAddress := r.URL.Query().Get("q")
```

Comment:
This method handles both federation and reverse federation requests. So changing this variable name to `stellarAddress` can be misleading.